README.md

```diff
@@ -21,6 +21,7 @@ $ cd dns
 $ virtualenv env
 ...
 $ source env/bin/activate
+$ pip install -U setuptools
 $ pip install octodns
```

Comment:
What's the error message people will get if they try and install octoDNS without updating setuptools? I'd imagine people familiar with python are just gonna type `pip install octodns` and I like to avoid that being an immediate failure. I guess that's my biggest reservation with making this update/change in general. 

Beyond that I think i'd prefer to have something like `pip install setuptools>=<min-required-version>` here so that it doesn't imply that you always have to be running the absolute latest. That should allow people in situations who already install a specific version of setuptools to know whether or not they have something new enough.